### PR TITLE
TT-8937 Added edp sqlite persistent volume

### PR DESCRIPTION
**Pull request by rewsmith**
Added a new volume mount to enable the enterprise developer portal to be installed/configured with sqlite

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If PRing from your fork, don't come from your `master`!
- [ ] Make sure you are making a pull request against our **`master` branch** (left side). Also, it would be best if you started *your change* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: tyk-pro/templates/statefulset-enterprise-portal.yaml [hunk 1]
```diff
@@ -138,6 +138,9 @@ spec:
           - name: enterprise-portal-pvc-{{ include "tyk-pro.fullname" . }}
             mountPath: /opt/portal/themes
             subPath: themes
+          - name: enterprise-portal-pvc-{{ include "tyk-pro.fullname" . }}
+            mountPath: /opt/portal/db
+            subPath: db
```
Comment (caroltyk) [inline tyk-pro/templates/statefulset-enterprise-portal.yaml:143]: @rewsmith should this by applied only for sqlite?

Comment (rewsmith) [reply tyk-pro/templates/statefulset-enterprise-portal.yaml:143]: That's a good point @caroltyk.  Let me check and amend as required.

Comment (rewsmith) [reply tyk-pro/templates/statefulset-enterprise-portal.yaml:144]: Changes made and tested @caroltyk

**Review by andrei-tyk — APPROVED** [review]

**Review by zalbiraw — APPROVED** [review]
LGTM

**Pull request merged**